tensorflow/lite/kernels/internal/reference/strided_stride.h

```diff
@@ -0,0 +1,80 @@
+/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
file should be strided_slice not strided_stride i think ?

Reply:
@karimnosseir , thanks for the review, i have renamed the file.

Regards
Amit